dev_guide/deployments/advanced_deployment_strategies.adoc

```diff
@@ -28,51 +30,49 @@ endif::[]
+(the blue version). You can use a
+xref:../../dev_guide/deployments/deployment_strategies.adoc#rolling-strategy[rolling
+strategy] or switch services in a route.
 
-Use a xref:dev_guide/deployments/deployment_strategies.adoc#[blue-green deployment]
-when you want to switch users to a new, blue, version of your application in
-a production, green, environment.
+[NOTE]
+====
+Since many applications depend on persistent data, you will need to have an
+application that supports xref:n1-compatibility[N-1 compatibility] if you share
+a database, or implement a live data migration between your database, store, or
+disk if you choose to create two copies of your data layer.
 
-Blue-green deployments make switching between two different versions of your
-application easy. However, since many applications depend on persistent data,
-you will need to have an application that supports xref:n1-compatibility[N-1
-compatibility] if you share a database, or implement a live data migration
-between your database, store, or disk if you choose to create two copies of your
-data layer.
+Consider the data used in testing the new version. If it is the production data
+set a bug in the new version can break the production version.
+====
 
-Consider the data used in testing the new version. If it is the production data set
-a bug in the new version can break the production version.
 
 [[advanced-deployment-strategies-blue-green-deployments-example]]
-=== Blue-Green Deployment Example
+=== Using a Blue-Green Deployment
 
-This strategy uses two deployment configurations. One, green, is for the current,
-production, version and the other is for the new, blue, version. Each deployment
-configuration is exposed to a different service.
+Blue-Green deployment use two deployment configurations. Both are running, and
+the one that is used depends on the service the route specifies, with each
+deployment configuration exposed to a different service. You can create a new
+route to the new version and test it. When ready, change the service in the
+production route to point to the new service and the new, blue, version is live.
 
-Both deployments are up and running and the one that is used depends on 
-the service the route specifies. you can create a new route to the new version and 
-test it. When ready, change the service in the production route to point to the new 
-service and the new, blue, version is live. If necessary, you can roll back to the old,
-green, version by switching service back to the previous version.
+If necessary, you can roll back to the older, green, version by switching
+service back to the previous version.
 
+[discrete]
 [[advanced-deployment-strategies-using-a-route-and-two-services]]
-=== Using a Route and Two Services
+==== Using a Route and Two Services
+
+This example sets up two deployment configurations; one for the stable, green,
+version and the other for the newer, blue, version. 
```

Comment:
Maybe "the stable version (green) and the other for the newer version (blue). A bit awkward as is, to me anyway.